docs/en/wallet-solution.rst

```diff
@@ -62,6 +62,203 @@ Deactivation
+
+Payload
+^^^^^^^
++-----------------------------------+-----------------------------------+
+| **Key**                           | **Value**                         |
++-----------------------------------+-----------------------------------+
+| iss                               | Public URL of the Wallet          |
+|                                   | Provider.                         |
++-----------------------------------+-----------------------------------+
+| sub                               | Public URL of the Wallet          |
+|                                   | Provider.                         |
++-----------------------------------+-----------------------------------+
+| iat                               | Issuance datetime in              |
+|                                   |  Unix Timestamp format.           |
++-----------------------------------+-----------------------------------+
+| exp                               | Expiration datetime               |
+|                                   | in Unix Timestamp format.         |
++-----------------------------------+-----------------------------------+
+| jwks                              | Contains an array of all public   |
+|                                   | keys associated with the domain.  |
+|                                   | These could match the Wallet      |
+|                                   | Provider's keys.                  |
++-----------------------------------+-----------------------------------+
+| metadata                          | For each entity, this attribute   |
+|                                   | houses its metadata. In this case,|
+|                                   | it contains the Wallet Provider's |
+|                                   | metadata within the               |
+|                                   | ``eudi_wallet_provider`` attribute|
+|                                   | and the generic entity            |
+|                                   | ``federation_entity``.            |
++-----------------------------------+-----------------------------------+
+
+Payload `wallet_provider`
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
++------------------------------------+------------------------------------+
+| **Key**                            | **Value**                          |
++------------------------------------+------------------------------------+
+| jwks                               | Contains an array of all the Wallet|
+|                                    | Provider's public keys.            |
++------------------------------------+------------------------------------+
+| token_endpoint                     | Endpoint for obtaining the Wallet  |
+|                                    | Instance Attestation.              |
++------------------------------------+------------------------------------+
+| attested_security_context          | List of supported values for the   |
```

Comment:
```suggestion
| attested_security_context_values_supported          | List of supported values for the   |
```

Reply:
this will break the linting but I merge your suggestion anyway then I'll fix it in a local branch then a I'll do another commit